plugins/ps/functions

```diff
@@ -194,7 +194,16 @@ ps_scale() {
   local DOKKU_PROCFILE="$DOKKU_ROOT/$APP/DOKKU_PROCFILE"
   shift 1
 
-  [[ ! -e $DOKKU_SCALE_FILE ]] && generate_scale_file "$APP" "$IMAGE_TAG" >/dev/null 2>&1
+  [[ ! -e $DOKKU_PROCFILE ]] && extract_procfile "$APP" "$IMAGE_TAG"
+  if [[ ! -e $DOKKU_SCALE_FILE ]] ; then
+    generate_scale_file "$APP" "$IMAGE_TAG"
```

Comment:
The output here should be redirected as before, right?

Reply:
Yes, fixed.